source/sync/get-started.txt

```diff
@@ -183,4 +183,9 @@ To create queries from your client application, refer to the Realm SDK documenta
 - :ref:`Flexible Sync - React Native SDK <react-native-flexible-sync>`
 - :ref:`Flexible Sync - Swift SDK <ios-flexible-sync>`
 
+.. warning:: Query Size Limit
+
+  The **size limit** for queries from your client application when using Device Sync is
+  **256 kB**. Exceeding this limit in a query will result in a :ref:`LimitsExceeded Error<flexible-sync-errors>`.
```

Comment:
Minor nit: our style guide calls for us to avoid future tense.
```suggestion
  **256 kB**. Exceeding this limit in a query results in in a :ref:`LimitsExceeded Error<flexible-sync-errors>`.
```